test/test_unit_credential_store_factory.py

```diff
@@ -0,0 +1,31 @@
+# Builtin
+import unittest
+from unittest.mock import patch, call
+
+from conjur.logic.credential_provider import CredentialStoreFactory, KeystoreCredentialsProvider, \
+    FileCredentialsProvider
+from conjur.wrapper import KeystoreAdapter
+
+
+class CredentialStoreFactoryTest(unittest.TestCase):
+    @patch.object(KeystoreAdapter, "get_keyring_name", return_value="SecretService Keyring")
+    @patch.object(KeystoreAdapter, "is_keyring_accessible", return_value=True)
+    def test_create_credential_store_returns_keystore_credentials_provider_when_there_is_keyring_and_its_accessible(
```

Comment:
```suggestion
    def test_create_credential_store_returns_keystore_when_there_is_keyring_and_is_accessible(
```
so that it is shorter. If you agree, let's make this change in all the test names removing `credentials_provider` from their names

Reply:
Removed credential_provider from func names